sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/Ecdsa.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.cryptography;
+
+import java.security.GeneralSecurityException;
+import java.security.KeyPair;
+import java.security.Provider;
+import java.security.Signature;
+
+abstract class Ecdsa extends AsymmetricSignatureAlgorithm {
+
+    protected Ecdsa() {
+        super("NONEwithEDCSA");
```

Comment:
Is it ECDSA or EDCSA?

Reply:
ECDSA

---
In reply to: [309923823](https://github.com/Azure/azure-sdk-for-java/pull/4667#discussion_r309923823) [](ancestors = 309923823)

Reply:
yeah, typo, fixed it.